examples/09_config_inside_rule_impl/defs.bzl

```diff
@@ -0,0 +1,13 @@
+load("@bazel_skylib//rules:common_settings.bzl", "BuildSettingInfo")
+
+def _impl(ctx):
+    print("Printing from " + ctx.attr.name)
+    print("foo_enabled = " + str(ctx.attr.foo_enabled[BuildSettingInfo].value))
+    return []
+
+my_rule = rule(
+    implementation = _impl,
+    attrs = {
+        "foo_enabled": attr.label(default = Label("//examples/09_config_inside_rule_impl:foo_enabled_flag"))
```

Comment:
This is a really quick pass, but does this rule utilize making this a user-overrideable attribute ("foo_enabled") vs. implicit ("_foo_enabled")?

Reply:
Great point. Made that change and added a note about it in the readme of this example.